### PR TITLE
clang-format: exhaustive style list

### PR DESCRIPTION
**Pull request by BuonOmo**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
Welcome to tldr @BuonOmo, The changes LGTM. Thanks for your contribution.

**Comment by CleanMachine1** [comment]
While we should definitely add the GNU and the microsoft, I think both links should have the exhaustive list so that the user doesn't have to read the other command examples for context.

File: pages/common/clang-format.md [hunk 1]
```diff
@@ -13,12 +13,12 @@
 
 - Format a file using a predefined coding style:
 
-`clang-format --style={{LLVM|Google|Chromium|Mozilla|WebKit}} {{path/to/file}}`
+`clang-format --style={{LLVM|GNU|Google|Chromium|Microsoft|Mozilla|WebKit}} {{path/to/file}}`
```
Comment (EmilySeville7cfg) [inline pages/common/clang-format.md:16]: There are too many alternatives here. I think we should define the maximum limit of alternative placeholder count not to allow very long commands.

File: pages/common/clang-format.md [hunk 2]
```diff
@@ -13,12 +13,12 @@
 
 - Format a file using a predefined coding style:
```
Comment (EmilySeville7cfg) [inline pages/common/clang-format.md:14]:
```suggestion
- Format a file using a specific predefined coding style:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/clang-format.md (hunk 1 above)
Comment (BuonOmo) [reply pages/common/clang-format.md:16]: @EmilySeville7cfg those changes could be nice as well but I don't think they are fully necessary, and I am definitely not the author of those. I don't file like it should be done in this PR or even by me, hence I am not sure why you _requested changes_ here?

Comment (EmilySeville7cfg) [reply pages/common/clang-format.md:16]: I've requested changes because another my comment requires them.

Comment (BuonOmo) [reply pages/common/clang-format.md:16]: @EmilySeville7cfg I guess you mention https://github.com/tldr-pages/tldr/pull/8493#discussion_r973734282.. My answer would be the same: my pull request is about fixing the list and I don't think I should be responsible I should do any other change in this PR.

Comment (EmilySeville7cfg) [reply pages/common/clang-format.md:16]: So I am going to resolve this conversation.

File: pages/common/clang-format.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/common/clang-format.md:14]: This change must be done in another PR by somebody else. I approve this PR with keep this note in mind.

**Review by EmilySeville7cfg — APPROVED** [review]
Approved, but requires changes in other PRs. See my comments for details.

**Pull request merged**